File: src/main/java/com/gdgswu/planeat/domain/auth/dto/request/SignupRequest.java
package com.gdgswu.planeat.domain.auth.dto.request;

import lombok.Builder;
import lombok.Getter;

import java.util.Set;

@Getter
@Builder
public class SignupRequest {
    private String idToken;

    private String name;
    private String gender;
    private int age;
    private int height;
    private int weight;
    private String location;

    private int mealsPerDay;
    private int hungerCycle;
    private Boolean canCook;
    private Set<Long> allergicFoodIds;
}

File: frontend/app/src/main/java/com/example/planeat/data/remote/dto/SignupRequest.kt
package com.example.planeat.data.remote.dto

data class SignupRequest(
    val idToken: String,
    val name: String,
    val gender: String,
    val age: Int,
    val height: Int,
    val weight: Int,
    val location: String,
    val mealsPerDay: Int,
    val hungerCycle: Int,
    val canCook: Boolean,
    val intolerances: List<String>
)